scripts/build_node_package.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+MODULE_PATH=/tmp/module
+BUILD_TARGET_DIR=/tmp/module_build
+
+git clone https://github.com/mmarchini-oss/node-linux-perf.git $MODULE_PATH
+cd $MODULE_PATH
+git reset --hard 20eb88a35ab256313dfb0f14645456ebf046ac1b
+
+npm install -g node-gyp
+curl -L https://github.com/nodejs/nan/archive/refs/tags/v2.16.0.tar.gz -o nan.tar.gz
+tar -vxzf nan.tar.gz -C /tmp
+NAN_PATH=$(realpath /tmp/nan-*)
+export NAN_PATH
+sed -i 's/node \-e \\"require('\''nan'\'')\\"/echo $NAN_PATH/g' binding.gyp
+rm -rf nan.tar.gz
+mkdir $BUILD_TARGET_DIR
+node_versions=( "10.10.0" "11.0.0" )
+for node_version in "${node_versions[@]}"; do
+    node-gyp configure --target="$node_version"  --build_v8_with_gn=false
+    node-gyp build --target="$node_version"
+    t=("${node_version//./ }")
```

Comment:
This gets messed up on 10 & 11, I see these values:
```
gprofiler/resources/node/module/glibc/11 0 0
gprofiler/resources/node/module/glibc/11 0 0/linux-perf.js
gprofiler/resources/node/module/glibc/11 0 0/build
gprofiler/resources/node/module/glibc/11 0 0/build/Release
gprofiler/resources/node/module/glibc/11 0 0/build/Release/linux-perf.node
gprofiler/resources/node/module/glibc/14
gprofiler/resources/node/module/glibc/14/linux-perf.js
gprofiler/resources/node/module/glibc/14/build
gprofiler/resources/node/module/glibc/14/build/Release
gprofiler/resources/node/module/glibc/14/build/Release/linux-perf.node
gprofiler/resources/node/module/glibc/10 10 0
gprofiler/resources/node/module/glibc/10 10 0/linux-perf.js
gprofiler/resources/node/module/glibc/10 10 0/build
gprofiler/resources/node/module/glibc/10 10 0/build/Release
gprofiler/resources/node/module/glibc/10 10 0/build/Release/linux-perf.node
```

Reply:
(I expect it to be just `11` for 11, not `11 0 0`, no? Otherwise the loading code won't find it, because it searches for the major only)

Reply:
Looks good now:
```
/app/gprofiler/resources/node/module/glibc/20eb88a
/app/gprofiler/resources/node/module/glibc/20eb88a/11
/app/gprofiler/resources/node/module/glibc/20eb88a/11/linux-perf.js
/app/gprofiler/resources/node/module/glibc/20eb88a/11/build
/app/gprofiler/resources/node/module/glibc/20eb88a/11/build/Release
/app/gprofiler/resources/node/module/glibc/20eb88a/11/build/Release/linux-perf.node
/app/gprofiler/resources/node/module/glibc/20eb88a/10
/app/gprofiler/resources/node/module/glibc/20eb88a/10/linux-perf.js
/app/gprofiler/resources/node/module/glibc/20eb88a/10/build
/app/gprofiler/resources/node/module/glibc/20eb88a/10/build/Release
/app/gprofiler/resources/node/module/glibc/20eb88a/10/build/Release/linux-perf.node
/app/gprofiler/resources/node/module/glibc/20eb88a/12
/app/gprofiler/resources/node/module/glibc/20eb88a/12/linux-perf.js
/app/gprofiler/resources/node/module/glibc/20eb88a/12/build
/app/gprofiler/resources/node/module/glibc/20eb88a/12/build/Release
/app/gprofiler/resources/node/module/glibc/20eb88a/12/build/Release/linux-perf.node
/app/gprofiler/resources/node/module/glibc/20eb88a/13
/app/gprofiler/resources/node/module/glibc/20eb88a/13/linux-perf.js
/app/gprofiler/resources/node/module/glibc/20eb88a/13/build
/app/gprofiler/resources/node/module/glibc/20eb88a/13/build/Release
/app/gprofiler/resources/node/module/glibc/20eb88a/13/build/Release/linux-perf.node
/app/gprofiler/resources/node/module/glibc/20eb88a/14
/app/gprofiler/resources/node/module/glibc/20eb88a/14/linux-perf.js
/app/gprofiler/resources/node/module/glibc/20eb88a/14/build
/app/gprofiler/resources/node/module/glibc/20eb88a/14/build/Release
/app/gprofiler/resources/node/module/glibc/20eb88a/14/build/Release/linux-perf.node
/app/gprofiler/resources/node/module/glibc/20eb88a/15
/app/gprofiler/resources/node/module/glibc/20eb88a/15/linux-perf.js
/app/gprofiler/resources/node/module/glibc/20eb88a/15/build
/app/gprofiler/resources/node/module/glibc/20eb88a/15/build/Release
/app/gprofiler/resources/node/module/glibc/20eb88a/15/build/Release/linux-perf.node
/app/gprofiler/resources/node/module/glibc/20eb88a/16
/app/gprofiler/resources/node/module/glibc/20eb88a/16/linux-perf.js
/app/gprofiler/resources/node/module/glibc/20eb88a/16/build
/app/gprofiler/resources/node/module/glibc/20eb88a/16/build/Release
/app/gprofiler/resources/node/module/glibc/20eb88a/16/build/Release/linux-perf.node
```